### ISSUE TITLE
Archlinux guide outdated

### ISSUE THREAD

**Issue by unman** [Qubes-Community/Contents#180]
As QubesOS/qubes-issues#7401 shows, the guide is broken for 4.1 - at a minimum delete the reference to 99-makeflags.sh

More than that, it isnt clear to me why there needs to be a separate guide for building Arch. I really dont see the point.
Why not fix up the official docs (if they are inadequate), and raise issues against the builder if issues exist?
This just looks like a hostage to fortune.

**Comment by awokd** [comment]
I think the document is a holdover from 3.x(or earlier?) when the builder didn't natively support Archlinux templates. If it does now, and the only issue with it is adding a `make remount` step somewhere as you mention in https://github.com/QubesOS/qubes-issues/issues/7401#issuecomment-1091835819, agree consolidating it seems like the best approach. @lubellier , thoughts?

**Comment by lubellier** [comment]
This guide helped me to build my first QubesOS-4.0 packages, and my first packages was ArchLinux packages so I used this guide. Now I use it for QubesOS-4.1, and I continue to build my ArchLinux packages and templates. I follow the steps, but without the step-1 about the fedora-minimal AppVM, I use a standard fedora-34 AppVM. The guide contains also specific ArchLinux topics (build a repository, ...). In the past I updated this guide, added a guide status header and helped others users to update it also.

So for me this guide is usefull. But I agree a good documentation is a live documentation and should be updated.

So I will read the [official guides](https://www.qubes-os.org/doc/#building), see how to use the official guides for ArchLinux, and try to reduce the content of the building-archlinux-template.md. When the building-archlinux-template.md will be "just read the official guides", we will delete it.
